src/main/java/org/jabref/logic/util/io/BackupFileUtil.java

```diff
@@ -0,0 +1,122 @@
+package org.jabref.logic.util.io;
+
+import java.io.IOException;
+import java.nio.ByteBuffer;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.time.ZonedDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.HexFormat;
+import java.util.Optional;
+
+import org.jabref.logic.util.BackupFileType;
+import org.jabref.logic.util.BuildInfo;
+
+import net.harawata.appdirs.AppDirsFactory;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class BackupFileUtil {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(BackupFileUtil.class);
+
+    private BackupFileUtil() {
+    }
+
+    public static Path getAppDataBackupDir() {
+        Path directory = Path.of(AppDirsFactory.getInstance().getUserDataDir(
+                                     "jabref",
```

Comment:
We already have a directory in appdir called "JabRef". This yields a second directory. Maybe use the constant SEARCH_INDEX_BASE_PATH from BibDatabaseContext and rename it.